cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -57,7 +59,7 @@
   },
   { "name": "varlogetcd",
     "hostPath": {
-        "path": "/var/log/etcd.log"}
+        "path": "/var/log/etcd{{ suffix }}.log"}
```

Comment:
I think that we should also change the "varetcd" volume.
I think that the path should be:
"/mnt/master-pd/var/etcd{{ suffix }}"

Otherwise, we would have a conflict that two separate etcd instances would be writing to the same directory - it seems bad to me.